selfdrive/car/tests/test_common.py

```diff
@@ -0,0 +1,77 @@
+import unittest
+
+from selfdrive.car import common_fault_avoidance
+
+class TestCommonFaultAvoidance(unittest.TestCase):
```

Comment:
we already test all this specific behavior in panda, perhaps we should improve our fuzzy test cases in test_models for this

Reply:
If there's any test case here that you noticed we don't test in panda, we should add it there @jnewb1

Reply:
Seems like the panda safety already checks all these scenarios, I'll go ahead and remove the tests.